dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/ConditionsTaskTest.java

```diff
@@ -17,7 +17,16 @@
 package org.apache.dolphinscheduler.server.master;
```

Comment:
Hi,
I checkout to your branch and found that the `ConditionsTaskTest.java` is not in the test case includer.
Please add the test to root pom.
![image](https://user-images.githubusercontent.com/29545877/89162341-8328d480-d5a6-11ea-9fec-d895d57e44cc.png)